config/prod/amp-ad-rna-seq-repro-rosmap-output-bucket.yaml

```diff
@@ -0,0 +1,46 @@
+# Provision a general S3 bucket or a Synapse External Bucket
+# http://docs.synapse.org/articles/custom_storage_location.html
+template_path: remote/s3-bucket.yaml
+stack_name: amp-ad-rna-seq-repro-rosmap-output
```

Comment:
[optional] would it make sense to add `temp` to the name? or at least add a comment that this contains temporary data?

Reply:
Did this.